### PR TITLE
Add a Setup Form for Creating and Editing Policies

### PR DESCRIPTION
**Pull request by thaorell**
## Description:
Continuation of #59

## Checklist:
 - [X] Related tests were updated
 - [X] Related documentation was updated

### CODE REVIEW

File: src/components/policies/Policies.tsx [hunk 1]
```diff
@@ -12,102 +12,90 @@ import {
...
   };
   useEffect(queryPolicy, [dispatch, token]);
-
+  useEffect(() => {
+    if (policyId) {
+      const currentPolicy = policyStore[policyId];
+      setDefaultValues(getDefaultFormValues(currentPolicy));
+    }
+  }, [policyId, policyStore]);
+  // let currentPolicy = policyStore[Number(policyId)];
```
Comment (brhatiga) [inline src/components/policies/Policies.tsx:72]: Is this comment still needed?

File: __tests__/components/QuickClusterWizard/QuickClusterWizard.test.tsx [hunk 2]
```diff
@@ -136,7 +136,7 @@ describe('<QuickClusterWizard />', () => {
 
     // Closes the window
     expect(history.action).toBe('POP');
-  });
+  }, 10000);
```
Comment (brhatiga) [inline __tests__/components/QuickClusterWizard/QuickClusterWizard.test.tsx:139]: Just curious, why 10000?

Comment (thaorell) [reply __tests__/components/QuickClusterWizard/QuickClusterWizard.test.tsx:139]: @brhatiga the default 5000ms would sometimes result in timeout and failure if there isn't enough memory.

Comment (brhatiga) [reply __tests__/components/QuickClusterWizard/QuickClusterWizard.test.tsx:139]: @thaorell ah, good information.  Would you mind adding it as a comment?

File: src/components/policies/Policies.tsx (hunk 1 above)
Comment (thaorell) [reply src/components/policies/Policies.tsx]: Nope. Will remove!

**Review by brhatiga — APPROVED** [review]

**Pull request merged**